compiler/rustc_apfloat/src/lib.rs

```diff
@@ -30,7 +30,7 @@
 //!
 //! This API is completely unstable and subject to change.
 
-#![doc(html_root_url = "https://doc.rust-lang.org/nightly/")]
+#![doc(html_root_url = "https://doc.rust-lang.org/nightly-rustc/")]
```

Comment:
All of them should be:
```suggestion
#![doc(html_root_url = "https://doc.rust-lang.org/nightly/nightly-rustc/")]
```

Reply:
I feel _very_ stupid now. I'll try that again.